### ISSUE TITLE
Color contrast on dark theme show invisible link labels

### ISSUE THREAD

**Issue by nickofthyme** [elastic/elastic-charts#1499]
**Describe the bug**
When using the default background color on the dark theme, or black background, the `linkLabel` on a partition chart should be white or at least visible. 

![image](https://user-images.githubusercontent.com/19007109/142929082-d96ede34-e103-42ee-8c66-77a5762c664a.png)
[photo link](https://github.com/elastic/elastic-charts/blob/16246550d35810ef7fb6bcf9ccf4118b4d876683/integration/tests/__image_snapshots__/partition-stories-test-ts-axis-stories-theme-dark-should-show-default-text-color-in-dark-theme-1-snap.png)

**To Reproduce**
1. Go to [`24_linked_label.story.tsx`](https://github.com/elastic/elastic-charts/blob/master/storybook/stories/sunburst/24_linked_label.story.tsx)
2. Apply no `textColor` to the `linkLabel` so it uses the default value
3. Change theme to `dark` (not `eui-dark`)
4. Notice the color contrast logic is bad rendering invisible (black) link labels
5. Changing the background color to black causes the same result.

**Expected behaviour**
Color contrast logic returns a sensible color with black background.
